Sources/TuistScaffold/TemplateGenerator.swift

```diff
@@ -107,11 +107,19 @@ public final class TemplateGenerator: TemplateGenerating {
                 } else {
                     renderedContents = fileContents
                 }
+            case let .directory(path):
+                let destinationContainerPath = destinationPath.appending(components: [$0.path.pathString, path.basename])
```

Comment:
I'd rename `destinationContainerPath` to `destinationDirectoryPath`.

Also nit: you can leverage the variadic arguments for `appending()` and rewrite `.appending(components: [x, y])` to `.appending(components: x, y)`

Reply:
cool, thanks